.github/workflows/ci.yml

```diff
@@ -197,6 +197,10 @@ jobs:
         working-directory: ${{runner.temp}}
         run: mkdir clerk-js && cd clerk-js && npm init -y && npm install @clerk/clerk-js
 
+      - name: Copy components @clerk/astro
+        if: ${{ matrix.test-name == 'astro' }}
+        run: cd packages/astro && npm run copy:components
+
```

Comment:
⚠ This is needed because the above command from Verdaccio `npx turbo build $TURBO_ARGS --only` does not seem to copy the files correctly.